### PR TITLE
[lliu41] Company API + EF core 

### PR DESCRIPTION
**Pull request by kristol07**

### CODE REVIEW

File: EFCoreRelationshipsPractice/Entites/CompanyEntity.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            this.Profile = new ProfileEntity(companyDto.Profile);
+            this.Employees = companyDto.Employees?
+                .Select(employeeDto => new EmployeeEntity(employeeDto))
+                .ToList();
+        }
+
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public ProfileEntity? Profile { get; set; }
+        [ForeignKey("CompanyId")]
```
Comment (YYuan1414) [inline EFCoreRelationshipsPractice/Entites/CompanyEntity.cs:28]: another way to add foreign key in the CompanyEntity. good

File: EFCoreRelationshipsPractice/Entites/EmployeeEntity.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        {
+        }
+
+        public EmployeeEntity(EmployeeDto employeeDto)
+        {
+            Name = employeeDto.Name;
+            Age = employeeDto.Age;
+        }
+
+        public int? Id { get; set; }
```
Comment (YYuan1414) [inline EFCoreRelationshipsPractice/Entites/EmployeeEntity.cs:22]: Why id can be null ?

Comment (kristol07) [reply EFCoreRelationshipsPractice/Entites/EmployeeEntity.cs:22]: > Why id can be null ?

agree, you are right.

**Comment by kristol07** [comment]
Backend Learning

**Pull request merged**